tests/testthat/test_features.R

```diff
@@ -371,7 +371,7 @@ test_that("computeFeatures works as intended", {
   expect_equal(nrow(dt), 4)
 
   # test from example table in orfik
-  dt <- computeFeatures(grl = grl,orfFeatures = T, RFP = RFP7, RNA = RNAGAlign,
+  dt <- computeFeaturesSpecial(grl = grl,orfFeatures = T, RFP = RFP7, RNA = RNAGAlign,
```

Comment:
name of the special will definitely not be special

Reply:
What about computeFeaturesCage ?
I will try to make the txdb maker function now, and see what we can do to remove this function.

Reply:
Mixing logic of reassigning start codons with calculating features just can not go together as those are completely two separate problems.